packages/plugins/rubygem-openscap_parser/rubygem-openscap_parser.spec

```diff
@@ -1,26 +1,29 @@
-# template: nonscl
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name openscap_parser
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 1.0.0
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Parse OpenSCAP content
 Group: Development/Languages
 License: MIT
-URL: https://github.com/OpenSCAP/openscap_parser
+URL: https://github.com/dLobatog/openscap_parser
```

Comment:
It was actually moved. Opened https://github.com/OpenSCAP/openscap_parser/pull/28 to reflect this.